tree/ntuple/v7/doc/specifications.md

```diff
@@ -282,6 +282,7 @@ The header consists of the following elements:
  - String: name of the ntuple
  - String: description of the ntuple
  - String: identifier of the library or program that writes the data
+ - 64bit unsigned integer: default compression settings
```

Comment:
Yes, I think that would be a good idea. I think it would also make sense to distinguish between "I don't know" and "multiple" if this is just for reporting purposes (the first one signaling that there's something going on we are not sure about, the second one signaling we do know what is going on).

Reply:
(See updated PR comment regarding compression settings)